README.md

```diff
@@ -92,9 +93,13 @@ implementation project(':java-library-glpi')
 
 ## Matrix
 
-|**GLPI Version**|9.1|9.2|9.3|
-|:----|----|----|----|
-|**GLPI API Client**||||
+|**Library**|**Android API**|
```

Comment:
You lost the format of the table.
How you can add a min and max values?
Could you provide an example?

Reply:
Here I switched the position of the version with the Android/Java column